migrations/851-fileupload-auto-increment-integer-pk.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `file_uploads` DROP PRIMARY KEY, ADD id INT PRIMARY KEY AUTO_INCREMENT, ADD UNIQUE (uuid);
```

Comment:
That's much simpler than I was expecting. Do you mind putting each change on its own line? I was surprised it ended up with an index on `uuid` since I didn't see it in here at first.

```sql
ALTER TABLE `file_uploads`
  DROP PRIMARY KEY,
  ADD id INT PRIMARY KEY AUTO_INCREMENT,
  ADD UNIQUE (uuid);